addon/components/rdf-input-fields/concept-scheme-multi-select-checkboxes.js

```diff
@@ -47,50 +47,66 @@ export default class RDFInputFieldsConceptSchemeMultiSelectCheckboxesComponent e
   }
 
   loadOptions() {
-    const store = this.args.formStore;
-    const { sourceGraph, metaGraph } = this.args.graphs;
     const path = this.args.field.rdflibPath;
```

Comment:
apparently this is not working when we use the getter method of the field model `.path`.

Will not look into this now